core/src/window_service.rs

```diff
@@ -377,6 +389,7 @@ impl WindowService {
             completed_data_sets_sender,
             retransmit_sender,
             outstanding_requests,
+            in_wen_restart,
```

Comment:
It will be enabled for as long as the validator is not restarted. When wen_restart is done, we will set wen_restart state to DONE and restart the validator, then in_wen_restart will be false and this behavior will be turned off.